s3cortxutils/s3confstore/s3confstore/cortx_s3_confstore.py

```diff
@@ -0,0 +1,134 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import argparse
+# from cortx.utils.conf_store import Conf
+from cortx.utils.conf_store import ConfStore
+from urllib.parse import urlparse
+import os.path
+import json
+import sys
+
```

Comment:
pls add one liner msg about this class like "This class is utitlity class used for storing/fetching values form cortx-py-utils' Config store"

Reply:
We will have it added later.. kind of urgent to have this code merged ASAP.

Reply:
sure. this is just to fix codacy warnings for python classes.
we can address them in upcoming PR as well